demos/demo_tricamera.py

```diff
@@ -56,9 +68,9 @@ def main():
 
         observations_timestamps_list.append(
             [
-                observation.cameras[0].time_stamp,
-                observation.cameras[1].time_stamp,
-                observation.cameras[2].time_stamp,
+                observation.cameras[0].timestamp,
```

Comment:
Ooh I just remembered we had some inconsistency with the time_stamp and timestamp. This looks nice :+1: